d3/hyper.js

```diff
@@ -208,6 +231,61 @@ function generateDisplayNames(treeData) {
     }, common.allChildren);
 }
 
+// Function to find a node with a given operatorId assuming left-deep assignment
```

Comment:
I don't think we should be making the "left-deep assignment" assumption

Reply:
Agreed.